qa/tasks/mgr/dashboard/helper.py

```diff
@@ -355,6 +355,9 @@ def assertError(self, code=None, component=None, detail=None):
         if detail:
             self.assertEqual(body['detail'], detail)
 
+    def assertSortedEqual(self, first, second):
```

Comment:
I added this method to test results coming from the backend API as JSON, so we can't really use a set. There shouldn't be duplicates.

In the test itself, I could just wrap the arguments with `sorted()` if we really just want to avoid adding this method.